browser/src/control/Control.UIManager.js

```diff
@@ -1710,12 +1710,17 @@ L.Control.UIManager = L.Control.extend({
 	enableTooltip: function(element) {
 		var elem = $(element);
 		if (window.mode.isDesktop()) {
-			elem.tooltip();
-			elem.on("mousedown", function() {
-				$('.ui-tooltip').fadeOut(function() {
-					$(this).remove();
+			if ($('.ui-tooltip').length <= 0) {
+				elem.tooltip();
+				elem.on("mousedown", function() {
+					$('.ui-tooltip').fadeOut(function() {
+						$(this).remove();
+					});
 				});
-			});
+			} else {
```

Comment:
I believe it's better to show the last tooltip (it might have newer information).
Just:
1. hide old if exists
2. show new one
Is it possible to do it that way?

Reply:
@eszkadev that's a good idea, but in practice and if it's makes less code, I would just remove/hide all because what are the odds of a user wanting to constantly see the tooltip? Just thinking out loud here but, I would rather not see multiple tooltips that actually make it harder to operate with the thing I'm trying to work with (be that the rotation widget, or maybe I'm just trying to look closely to a preview) and if the user really wants I assume the user will just move away and back in to see the tooltip.

Reply:
yes, exactly what I wanted to say: we should have only one tooltip: last one ->  when new tooltip appears: remove all old, insert one new